python_scripts/ensemble.py

```diff
@@ -273,7 +277,7 @@ def bootstrap_sample(x, y):
 )
 
 # %% [markdown]
-# Theoretically, 63.2% of the original data points of the original dataset will
+# Empirically, 63.2% of the original data points of the original dataset will
```

Comment:
I have to say I did not understand this change, maybe @lucyleeow can comment? This is a theoretical thing in the sense that it can be proven right (when n_samples goes to infinity) ? For example: https://stats.stackexchange.com/a/88994

Maybe stay more vague i.e. something like this: "it can be shown that around 63.2% of the ..."

Reply:
My original comment was more about how we estimate it. I might be mistaken on this but:

we currently estimate by doing `bootstrap` on a `large_n` of samples

I suggested that doing `large_n` of `bootstrap`'s, and then taking the average of the `large_n` of bootstraps was more intuitive to me.

But this is really a minor point. Either way is fine

Reply:
If I recall well what I intended to write is that when `n -> inf` then 63.2% of the data in the bootstrap sample are unique which I thought to be a theoretical thing and not an empirical one?

Reply:
> when n -> inf then 63.2% of the data in the bootstrap sample are unique 

This makes things clearer for me, but just my opinion.